Marlin/Configuration_adv.h

```diff
@@ -1201,7 +1201,11 @@
   #define FINE_MANUAL_MOVE 0.025    // (mm) Smallest manual move (< 0.1mm) applying to Z on most machines
   #if IS_ULTIPANEL
     #define MANUAL_E_MOVES_RELATIVE // Display extruder move distance rather than "position"
-    #define ULTIPANEL_FEEDMULTIPLY  // Encoder sets the feedrate multiplier on the Status Screen
+    #if ENABLED(STATUS_FLOW_INSTEAD_OF_FEEDRATE)
+      #define ULTIPANEL_FLOWMULTIPLY  // Encoder sets the feedrate multiplier on the Status Screen
```

Comment:
Is it correct 'feedrate'comment in ULTIPANEL_FLOWMULTIPLY define?

Reply:
You are right. This is mistype.

Reply:
Corrected